bld/test/ut/libpal/CMakeLists.txt

```diff
@@ -17,21 +17,21 @@ set(libpal_ut_include
 # Misc pal components to test
 #
 set(_MISC_PAL_UNITS_UNDER_TEST
-    new__pal_sk_posix
+    pal_sk_posix
```

Comment:
The git rename operations from commit 6c1bde36faed203fcd413d2d104c70819f90b296 is missing.
And therefore the cmake build fails.